src/NServiceBus.Core/Performance/ReceiveStatisticsFeature.cs

```diff
@@ -14,27 +11,35 @@ public ReceiveStatisticsFeature()
         }
         protected internal override void Setup(FeatureConfigurationContext context)
         {
-            context.Pipeline.Register("ReceivePerformanceDiagnosticsBehavior", typeof(ReceivePerformanceDiagnosticsBehavior), "Provides various performance counters for receive statistics");
+            var performanceDiagnosticsBehavior = new ReceivePerformanceDiagnosticsBehavior(context.Settings.LocalAddress());
+
+            context.Pipeline.Register("ReceivePerformanceDiagnosticsBehavior", _ => performanceDiagnosticsBehavior, "Provides various performance counters for receive statistics");
```

Comment:
Should we add an overload without the builder which just allows to pass in an instance?